syntax.php

```diff
@@ -98,6 +98,12 @@ function render($mode, &$renderer, $data) {
         } elseif ($mode == 'metadata') {
             if ($renderer->capture) $renderer->doc .= $meta;
             return true;
+        } elseif ($mode == 'odt') {
+            // prevent caching to ensure the included pages are always fresh
+            if ($nocache) $renderer->info['cache'] = false;
+
+            $renderer->doc .= $renderer->cdata ($xhtml);
```

Comment:
Doesn't `$renderer->cdata()` already update `$renderer->doc`? At least in the xhtml renderer, `$renderer->cdata()` has no return value.